apps/deploy-web/src/pages/deployments/[dseq]/index.tsx

```diff
@@ -1,19 +1,15 @@
 import { DeploymentDetail } from "@src/components/deployments/DeploymentDetail";
 
-type Props = {
-  dseq: string;
-};
-
-const DeploymentDetailPage: React.FunctionComponent<Props> = ({ dseq }) => {
-  return <DeploymentDetail dseq={dseq} />;
+const DeploymentDetailPage: React.FunctionComponent = () => {
+  return <DeploymentDetail />;
 };
 
 export default DeploymentDetailPage;
 
-export async function getServerSideProps({ params }) {
-  return {
-    props: {
-      dseq: params?.dseq
-    }
-  };
-}
+// export async function getServerSideProps({ params }) {
```

Comment:
Why remove that?

Reply:
This was done to deploy the frontend on a third-party vendor. This was causing issues with build. This won't be removed,  will revert it to previous state.